packages/mantine/src/components/collection/Collection.tsx

```diff
@@ -99,15 +100,32 @@ interface CollectionProps<T> extends DefaultProps<Selectors<typeof useStyles>> {
      * @default false
      */
     required?: boolean;
+    /**
+     * Label associated with a collection (title)
+     */
+    label?: string;
+    /**
+     * Description for the collection (under the label)
+     */
+    description?: string;
+    /**
+     * Error message if input is invalid
+     *
+     * @default 'Invalid Input'
+     */
+    error?: string;
```

Comment:
By extending `Omit<InputWrapperBaseProps, 'inputContainer' | 'inputWrapperOrder'>`, we wouldn't have to manually declare those. You can have a look at the [CodeEditor](https://github.com/coveo/plasma/blob/master/packages/mantine/src/components/code-editor/CodeEditor.tsx#L35-L36) to see how its done